corehq/toggles.py

```diff
@@ -220,14 +220,29 @@ def randomness_percent(self):
     def _get_identifier(self, item):
         return '{}:{}:{}'.format(self.namespaces, self.slug, item)
 
-    def enabled(self, item, **kwargs):
+    def enabled(self, item, namespace=Ellipsis):
+        if namespace == NAMESPACE_USER:
+            namespace = None  # because:
+            # StaticToggle.__init__(): self.namespaces = [None if n == NAMESPACE_USER else n for n in namespaces]
+
+        if set(self.namespaces) != set(ALL_NAMESPACES) and namespace is Ellipsis:
```

Comment:
nitpick: it would be more efficient to reverse these conditions so the sets are only created/compared if `namespace is Ellipsis`.